lib/output/utils.js

```diff
@@ -29,10 +29,7 @@ function newObjectInRealm(globalObject, object) {
 const wrapperSymbol = Symbol("wrapper");
 const implSymbol = Symbol("impl");
 const sameObjectCaches = Symbol("SameObject caches");
-const ctorRegistrySymbol = Symbol.for("[webidl2js]  constructor registry");
-
-// This only contains the intrinsic names that are referenced from the `ctorRegistry`:
-const intrinsicConstructors = ["Array"];
```

Comment:
Storing these in the constructor registry was done to correctly implement the parts of **WebIDL** that use intrinsic objects, and to allow using `extends ctorRegistry.<base‑name>` instead of `extends globalObject.<base‑name>`.

Reply:
I think we still implement the parts Web IDL that use intrinsic objects correctly. And I don't see the value of `extends ctorRegistry.` instead of `extend globalObject.`; indeed requiring everything to use the same ctorRegistry seems subpar instead of using the global as the coordination point.